docs/guide/faq.md

```diff
@@ -70,3 +74,66 @@ underlying cells with a call to `interpretAs`. If you trust your
 `CellType` and wish for its semantics to be preserved through
 transformation, use `convert`.  
 
+
+## How do I import GeoTrellis methods?
+
+Q: In some of the GeoTrellis sample code and certainly in example
+projects, it looks like some GeoTrellis types have more methods than
+they really do. If I create an `IntArrayTile`, it doesn't have most of
+the methods that it should - I can't reproject, resample, or carry out
+map algebra operations - why is that and how can I fix it?  
+
+A: Scala is a weird language. It is at once object oriented (obviously
```

Comment:
Semantic ambiguity with "at once". On first reading I read as "right away". How about "at the same time"?